superset-frontend/src/SqlLab/actions/sqlLab.js

```diff
@@ -1402,7 +1402,7 @@ export function popQuery(queryId) {
           dbId: queryData.database.id,
           schema: queryData.schema,
           sql: queryData.sql,
-          name: `Copy of ${queryData.tab_name}`,
+          title: t('Copy of %s', queryData.tab_name),
```

Comment:
What's the rationale for the key change here (and below)?

Reply:
whoops